### PR TITLE
Admin dashboard moderate buttons color change

### PR DESCRIPTION
**Pull request by giovannyg**
Issue: 

https://github.com/publiclab/plots2/issues/7083

Thanks!

### CODE REVIEW

**Comment by giovannyg** [comment]
@ananyaarun

I updated my pr description linking the issue.

File: app/assets/stylesheets/style.css [hunk 1]
```diff
@@ -471,12 +471,16 @@ div.note.moderated h4 {
...
 .note p.moderated .btn {
-  color: #940;
+  color: #940 !important;
   border-color: #eb8;
   background: none;
   font-size: smaller;
 }
 
+.note p.moderated .btn:hover {
+  background-color: rgba(238, 187, 136, 0.5)
```
Comment (tsparksh) [inline app/assets/stylesheets/style.css]:
```suggestion
  background-color: rgba(238, 187, 136, 0.5);
```

**Comment by tsparksh** [comment]
Hey @giovannyg , you should create a feature branch for the PR i.e., suppose you start working on a feature, then you can create a new branch from the master branch using git branch -b branch-name.

This will allow you to work on multiple features at once, otherwise, you have to wait for the previous PR to be merged so that master becomes free for the new branch and so on.

Please keep this in mind in the future.

**Comment by giovannyg** [comment]
Ok, I got it, thanks.

**Comment by tsparksh** [comment]
And please provide a screenshot if your PR has UI changes.

**Comment by nstjean** [comment]
@tsparksh This is a FTO, the screenshot is in the original issue. :)

FireShot Capture 159 - 🎈 Public Lab_ Dashboard - localhost

Thanks for catching the semicolon! @giovannyg can you make that change?

**Comment by giovannyg** [comment]
I created a new PR

https://github.com/publiclab/plots2/pull/7171

I didn't attached any screenshot cause I haven't set up the project in my computer. I hope this time everything is ok.

**Comment by jywarren** [comment]
Whoops, resolved a conflict, but should be all good now!!!

**Comment by jywarren** [comment]
Merged this as it was not a big deal, thanks so much!

**Pull request merged**